otx/algorithms/classification/adapters/mmcls/nncf/builder.py

```diff
@@ -80,7 +80,7 @@ def build_nncf_classifier(  # pylint: disable=too-many-locals,too-many-statement
                 dataset_builder=mmcls_build_dataset,
             ),
             config,
-            subset="train",
+            subset="val",
```

Comment:
```suggestion
            subset="train",
```
The quantisation parameter init dataset should be a train subset not a validation subset.

Reply:
We use train images, with "val" parameter  we construct batch differently, in the way we need for NNCF initialization

Reply:
ahh, sorry, my bad. it was for dataloader not dataset.